Sources/SwiftProtobuf/FieldNameMap.swift

```diff
@@ -8,6 +8,82 @@
 //
 // -----------------------------------------------------------------------------
 
+/// Proto field names are always ASCII, so we can improve the
+/// performance of some of our parsing by storing them as
+/// arrays of UInt8 to avoid the overhead of creating and
+/// comparing Unicode Strings.
```

Comment:
I went ahead and flipped the TextScanner over to build an `Array<UInt8>` instead of a `String` for the name lookup.  That was an easy >30% speedup there.  ;-)